salt/grains/iscsi.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+'''
+Grains for iSCSI Qualified Names (IQN).
+
+.. versionadded:: 2018.3.0
+
+To enable these grains set `iscsi_grains: True`.
+
+.. code-block:: yaml
+
+    iscsi_grains: True
+'''
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+import os
+
+# Import Salt libs
+import salt.modules.cmdmod
+import salt.utils.files
+import salt.utils.path
+import salt.utils.platform
+
+__proxyenabled__ = ['iscsi']
```

Comment:
Are these grains for proxy minions?

Reply:
I don't know, I was just copying other tests. I thought that was for alternate names or something.